README.md

```diff
@@ -1,26 +1,26 @@
 Baraja Doctrine database
 ========================
 
-Simple easy to use, maximal performance database layer with connection to Doctrine and support for Nette 3.0.
+A simple and easy to use, maximum performance database layer with connection to Doctrine, which allows you to use all the advantages of OOP and also has support for Nette 3.0.
 
-This package install Doctrine automatically to your project with stable run.
+This package automatically installs Doctrine (also setting everything up in the configuration) to your project and runs stably.
 
 **READY FOR NETTE 3.0!**
 
 How to install
 --------------
+This package can be installed using PackageRegistrator which is also part of the Baraja Sandbox(https://github.com/baraja-core/sandbox). If you are not using it, you have to install the package manually following this guide. 
```

Comment:
`PackageRegistrator` - Please add link to: https://github.com/baraja-core/package-manager